### ISSUE TITLE
Lazyloading images

### ISSUE THREAD

**Issue by henrikpalm** [glidejs/glide#362]
Hi i would like to propose a feature where you lazyload images so they only gets loaded if they are about to be shown. I guess it's possible to do manually with events and run.before but it would be nice to have just as a setup.
/h

**Comment by connorwilsonexceleron** [comment]
I don't know if this is still a relevant question, but I know in #2 it was mentioned that it wasn't really in the spirit of the project. I'm sure there is a hacky way to get [lazysizes](https://github.com/aFarkas/lazysizes) to work with it though

**Comment by alexandr-kazakov** [comment]
+1 Need lazyload

**Comment by tiagomatosweb** [comment]
+1

**Comment by bvdputte** [comment]
+1

**Comment by gregoirepuget** [comment]
+1

**Comment by joemidi** [comment]
+1

**Comment by wesley0609** [comment]
+1

**Comment by cstrouse** [comment]
For modern browsers you should be able to just add `loading="lazy"` to the images.

**Comment by henrikpalm** [comment]
Does that really work with hidden images that is in browser window. Because you dont want to load all images just because the slideshow is inscreen

**Comment by cstrouse** [comment]
Yes. https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading#Images_and_iframes

**Comment by bewards** [comment]
No, loading=lazy does not work. Per the web.dev documentation:


> Only images that are below the device viewport by the [calculated distance](https://web.dev/browser-level-image-lazy-loading/#distance-from-viewport-thresholds) load lazily. All images above the viewport, regardless of whether they're immediately visible, load normally.

**Comment by ericmorand** [comment]
Glide is a carousel. Why would it come with something specific to images?

**Comment by henrikpalm** [comment]
In my opinion lazy load is mandatory for carousels and it's perfectly fine to lazy load everything but the code and the text is difficult to stop from loading but images is easy to lazy load and they are also the bulk of the data.

**Comment by alexisRippstein** [comment]
+1

**Comment by dsacramone** [comment]
Any chance. this is something that is gonna be implemented?

**Comment by crolla97** [comment]
Opened a PR #641 to start tackling this issue but not sure how active maintenance is so may take a while
